features/steps/export_steps.py

```diff
@@ -1,15 +1,55 @@
 # Copyright (C) 2012-2021 jrnl contributors
 # License: https://www.gnu.org/licenses/gpl-3.0.html
 
+
 import json
 import os
 import shutil
 import random
 import string
 from xml.etree import ElementTree
-
+from colorama import Fore, Style
 from behave import given
 from behave import then
+import colorama
+
+
+def style_text(to_bold: bool, text_color: Fore, text_to_print: str):
+    """Generate colorized and styled text for expected output. Its purpose is the same as Entry.colorize
+
+    :param to_bold: Flag whether the text should be bolded
+    :type to_bold: bool
+    :param text_color: Valid colorama.Fore color for the text
+    :type text_color: colorama.Fore
+    :param text_to_print: Message contents
+    :type text_to_print: str
+    :return: Styled and colored output
+    :rtype: str
+    """
+    if to_bold:
+        text_style = Style.BRIGHT
+    else:
+        text_style = Style.NORMAL
+    text_color = getattr(colorama.Fore, text_color.upper(), None)
+    return text_style + text_color + text_to_print + Style.RESET_ALL
+
+
+@then("the output should be pretty printed")
+def check_export_pretty(context):
+    out = context.stdout_capture.getvalue()
+    lines = out.splitlines()
+
+    # As per the configuration,
+    expected_colorized_title = (
+        style_text(
+            True, context.jrnl_config["colors"]["date"].upper(), "2013-06-09 15:39"
```

Comment:
Just FYI, we generally don't like hardcoding test data into steps like this. We're currently rewriting the behave steps to pytest-bdd anyway, though, so this can stay.

Reply:
Thanks for pointing this out. So whatever tests that have this qualification step that don't also use pretty.yaml now fail. 

I think I'm just going to delete that step